modules/oe_theme_content_news/oe_theme_content_news.module

```diff
@@ -29,3 +31,16 @@ function oe_theme_content_news_preprocess_node__oe_news(&$variables) {
     $variables['content']['featured_image']['#attributes']['class'][] = 'ecl-u-type-paragraph';
   }
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_view_alter().
+ */
+function oe_theme_content_news_node_view_alter(array &$build, EntityInterface $entity, EntityViewDisplayInterface $display) {
+  if ($entity->bundle() === 'oe_news'
+    && $build['#view_mode'] === 'full'
+    && !$entity->get('oe_author')->isEmpty()
+    && $entity->get('oe_author')->count() > 1
+    && isset($build['oe_author']['#title'])) {
+    $build['oe_author']['#title'] = t('Authors');
+  }
```

Comment:
Let's use the same way we do on other alters, even if it's one field for now (more will come here):

```php
  if ($entity->bundle() !== 'oe_news') {
    return;
  }

  // Field labels are different in frontend and backend.
  // Here we override them, per view mode.
  if ($build['#view_mode'] === 'full') {
    $plural_label_overrides = [
      'oe_author' => t('Authors'),
    ];

    foreach ($plural_label_overrides as $name => $label_override) {
      if (!$entity->get($name)->isEmpty() && $entity->get($name)->count() > 1 && isset($build[$name]['#title'])) {
        $build[$name]['#title'] = $label_override;
      }
    }
  }
```